app/Http/Controllers/RoomController.php

```diff
@@ -19,6 +19,13 @@ public function index(Request $request)
         $bot = Bot::findOrFail($request->bot_id);
         $chatwork = Chatwork::withAPIToken($bot->bot_key);
         $rooms = $chatwork->rooms()->list();
-        return $rooms;
+        $group_boxs = [];
+        foreach($rooms as $room) {
+            if ($room['type'] == "group") {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Controllers/RoomController.php</br>**Line**: 24</br>String "group" does not require double quotes; use single quotes instead